### PR TITLE
added functionality next button and added design

### PR DESCRIPTION
**Pull request by ros-a**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,6 +19,7 @@ function get_new_question() {
         let current_movie = remaining_movies.pop()
         let quote_element = document.getElementById('quote')
         quote_element.textContent = current_movie.quote
+        console.log(current_movie.quote)
```
Comment (SamBellDev) [inline index.js]: No console logs should make it on to the codebase. Only for bug fixing

File: index.js [hunk 2]
```diff
@@ -46,6 +50,11 @@ document.getElementById('submit_button').addEventListener('click', function () {
     user_feedback_elem.style.display = 'block'
 })
 
+document.getElementById('next_question_button').addEventListener('click', function () {
+    get_new_question()
+    document.getElementById('next_question_button').disabled = true
```
Comment (SamBellDev) [inline index.js:57]: This may be the issue of bugs. Stops us being able to progress.

**Review by SamBellDev — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ros-a) [reply index.js]: done!

File: index.js (hunk 2 above)
Comment (ros-a) [reply index.js:57]: done!

**Review by SamBellDev — APPROVED** [review]
naicu

**Pull request merged**